src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -916,11 +916,11 @@ private IList<SqlDataRecord> CreateListOfRecords(StePermutation tvpPerm, object[
                     }
                     catch (OverflowException oe)
                     {
-                        Console.WriteLine("Failed Row[{0}]Col[{1}] = {2}: {3}", rowOrd, colOrd, row[colOrd], oe.Message);
+                        Console.WriteLine("Failed Row[{0}]Col[{1}] = {2}: {3}", rowOrd, colOrd, DateTimeVariantTest.GetValueString(row[colOrd]), oe.Message);
```

Comment:
Can we move these methods we need to call from 2 different classes to `DataTestUtility` instead, as calling method from another test class, doesn't seem right approach.

Reply:
I have moved `GetValueString` to DataTestUtility.